www/themes/custom/drupalcampcebu2015/public/sass/partials/_global.scss

```diff
@@ -112,6 +112,7 @@ a:hover {
 .header-info-top {
   a {
     color: $color-white-text;
+    text-decoration: underline;
   }
   a:hover {
     color: $color-white-text;
```

Comment:
I mean, what happen when you hover it ? does the underline disappears ? Should we not add one underline here too ?